### ISSUE TITLE
openitcockpit/victoria-metrics 4.6.9 and 4.6.10 not startup 

### ISSUE THREAD

**Issue by KayRitzmann** [it-novum/openITCOCKPIT-ce-docker#6]
run.sh runs in error:

unprocessed command-line args left: %s; the most likely reason is missing `=` between boolean flag name and value; see https://pkg.go.dev/flag#hdr-Command_line_flag_syntax

there was breaking change in issue:
https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4845

Solution could be:

- delete run.sh
- change in compose.yaml:
```
victoria-metrics:
    image: openitcockpit/victoria-metrics:latest
    env_file:
     - openitcockpit.env
    command:
     - "--storageDataPath=/storage"
     - "--graphiteListenAddr=:2003"
     - "--opentsdbListenAddr=:4242"
     - "--httpListenAddr=:8428"
    ports:
      - "8428"       # victoria-metrics
    networks:
      - oitc-backend
    deploy:
      restart_policy:
        condition: on-failure
```
Or use same approach in container with entry point and cmd

**Comment by nook24** [comment]
Hi @KayRitzmann thanks for reporting this. I will take a look at this.

**Comment by nook24** [comment]
I found the issue. openITCOCKPIT uses the official `victoriametrics/victoria-metrics` as base image. The `ENTRYPOINT` defined in the original image collided with our `CMD` command.

We have published a fixed version as nightly: `openitcockpit/victoria-metrics:nightly` and we are going to release a stable version soon as well.

**Comment by nook24** [comment]
This has been resolved with version 4.7.0: https://hub.docker.com/r/openitcockpit/openitcockpit-ce/tags?page=1&name=4.7.0

**Comment by nook24** [comment]
I'm closing this as it got resolved with the release of openITCOCKPIT 4.7.0 and the current version is 4.8.2 :)
